src/runtime/hexagon_remote/Makefile

```diff
@@ -41,6 +41,17 @@ COMMON_FLAGS = -I ${HEXAGON_SDK_INCLUDES}/stddef
 COMMON_CCFLAGS = ${COMMON_FLAGS} -O3 -I ${HEXAGON_SDK_LIBS}/common/remote/ship/android_Release
 HEXAGON_QAICFLAGS = ${COMMON_FLAGS}
 
+# rpcmem doesn't have a precompiled library in the SDK, so we just
```

Comment:
it seems like you figured it out. We should just push it thru and then do additional testing here on MTP using our toyCamera and HelloHexagon  as well as our usual tests.